src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -11,10 +11,12 @@ const MAX_REPOS_RETURNED = 20;
 
 export const GithubCreateBranchGet = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
 	const {
-		jiraHost,
+		jiraHost: jiraHostLocal,
 		githubToken,
 		gitHubAppConfig
 	} = res.locals;
+	const { jiraHost: jiraHostParam } = req.query;
+	const jiraHost = jiraHostLocal || jiraHostParam;
```

Comment:
We prioritize the session jiraHost over the param version, simply because we know the session one is always solid params should be but more opportunity to be wrong